.changelog/20348.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:new-service
```

Comment:
No need for a CHANGELOG entry when adding a new service - This file can be removed.

Reply:
Ah I see. Thanks for pointing that out. Removed it